docs/content/en/docs/pipeline-stages/builders/ko.md

```diff
@@ -389,7 +389,46 @@ To learn more about how Skaffold debugs Go applications, read the
 
 ### File sync
 
-File `sync` is not supported while the ko builder feature is in Alpha.
+The `ko` builder can
```

Comment:
Since this feature will only be available in V2, would it make sense to revert all doc changes here?

Reply:
We're not going to create any more v1.x releases from the `main` branch? I was thinking that this was perhaps the reason for the `docs` vs `docs-v2` split. If not, I'll revert the changes in `docs` 👍

Reply:
right now no. We need to clean up the docs and docs-v2 split. Depends on what @aaron-prindle has in mind. 
We do want the v1 and v2 docs to reside side by side so doc fixes could be made for users still using v1